### ISSUE TITLE
One Drive Hidden? 

### ISSUE THREAD

**Issue by GamesAndBacon** [aleksey-hoffman/sigma-file-manager#252]
## Feature request:

im not a one drive user. having the choice to hide it would be great :) 
possibly even support for other services? like google drive for example? idk how that would work if at all possible. 

thanks again :)

**Comment by WolvenSpectre** [comment]
Yes it is frustrating to having uninstalled One Drive to have it still be in the interface.
